configure

```diff
@@ -4,7 +4,6 @@
 #
 #
 # Copyright (C) 1992-1996, 1998-2012 Free Software Foundation, Inc.
-# Copyright IBM Corp. and others 2015
```

Comment:
We might want to keep this line.
Is the rest of this file automatically generated?

Reply:
Yes, this is the `configure` file that autoconf generates with the changes I've made to `configure.ac` and `omrmakefiles/confighelpers.m4`